magefile.go

```diff
@@ -41,7 +41,7 @@ import (
 
 const (
 	PKG       = "get.porter.sh/porter"
-	GoVersion = ">=1.17"
+	GoVersion = ">=1.18"
```

Comment:
If we only want to use Go 1.18 on the build server but allow people to use go 1.17 locally, then this line should be reverted.